### PR TITLE
Add downloads and follows to Projects list

### PR DESCRIPTION
**Pull request by joshieman06**
image

### CODE REVIEW

**Review by triphora — CHANGES_REQUESTED** [review]
Lint fails, also the dropdown should be fixed width
image

File: pages/dashboard/projects.vue [hunk 1]
```diff
@@ -419,6 +435,28 @@ export default defineNuxtComponent({
...
+        case 'Downloads':
+          sortedArray = projects.slice().sort((a, b) => {
+            if (a.downloads < b.downloads) {
+              return -1
+            }
+            if (a.downloads > b.downloads) {
+              return 1
+            }
+            return 0
+          })
```
Comment (triphora) [inline pages/dashboard/projects.vue:447]:
```suggestion
          sortedArray = projects.slice().sort((a, b) => a.downloads - b.downloads)
```

File: pages/dashboard/projects.vue [hunk 2]
```diff
@@ -419,6 +435,28 @@ export default defineNuxtComponent({
...
+        case 'Followers':
+          sortedArray = projects.slice().sort((a, b) => {
+            if (a.followers < b.followers) {
+              return -1
+            }
+            if (a.followers > b.followers) {
+              return 1
+            }
+            return 0
+          })
```
Comment (triphora) [inline pages/dashboard/projects.vue:458]:
```suggestion
          sortedArray = projects.slice().sort((a, b) => a.followers - b.followers)
```

File: pages/dashboard/projects.vue (hunk 1 above)
Comment (triphora) [reply pages/dashboard/projects.vue:447]: May not be lint-safe - make sure you still run `pnpm run fix`

File: pages/dashboard/projects.vue [hunk 3]
```diff
@@ -630,7 +672,7 @@ export default defineNuxtComponent({
   flex: 1;
   display: flex;
   flex-direction: row;
-  min-width: 0;
+  min-width: 230px;
```
Comment (triphora) [inline pages/dashboard/projects.vue:675]: Avoid px, try 15rem

**Review by triphora — COMMENTED** [review]
Should probably have a more elegant solution for smaller width screens
image

**Comment by triphora** [comment]
Will soon be closed if above comments are not addressed

**Comment by joshieman06** [comment]
School has started back up so I don't have time to engineer a fix for smaller screens. All other problems should be fixed

**Comment by Prospector** [comment]
Will close for now since mobile support is necessary

**Pull request closed without merging**